README.md

```diff
@@ -31,10 +33,6 @@ cd your-repo
 ### 2. Install Dependencies
 
 ```sh
-cd backend
```

Comment:
I understand that the backend and frontend are seperate packages, and I think it's good, but since they are both in the same repository and using a single README, and since the entire repository is already a package in itself (it has a `package.json`), I'd sort of expect to be able to `npm install` (or `build`, not sure which one is more appropriate) in the root of the repo, and have that action call `npm install` on both sub-packages (maybe even do it concurrently, but don't sweat it if it's non-obvious).